watir-rails.gemspec

```diff
@@ -15,9 +15,10 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license       = "MIT"
   gem.version       = Watir::Rails::VERSION
+  gem.required_ruby_version = ">= 2.2.0"
```

Comment:
Didn't know the existance of this property. Nice :) Can you explain your thought about `>= 2.2.0`? If going below it then Rails 3.x stops working?

Reply:
I couldn't figure out how to make it work as described at https://github.com/watir/watir-rails/issues/28. I _think_ that it could be made to work after https://github.com/rubygems/rubygems.org/pull/2590 is merged, but right now it would be a bunch of effort (I don't know what could break with 2.1 at the end of the series and I should at least retest every commit, which is tedious since test matrix would be expanded in the next patches).
I can do this if you think that retaining 2.1 support worth the effort, but I doubt that anyone on 2.1 would update wati-rails.